ethereum_bridge/src/protocol/transactions/votes/update.rs

```diff
@@ -213,42 +212,81 @@ mod tests {
     use namada_core::ledger::storage::testing::TestWlStorage;
     use namada_core::types::address;
     use namada_core::types::ethereum_events::EthereumEvent;
+    use namada_core::types::voting_power::FractionalVotingPower;
 
+    use self::helpers::{default_event, default_total_stake, TallyParams};
     use super::*;
-    use crate::protocol::transactions::votes::update::tests::helpers::{
-        arbitrary_event, setup_tally,
-    };
     use crate::protocol::transactions::votes::{self, EpochedVotingPower};
+    use crate::test_utils;
 
     mod helpers {
+        use namada_proof_of_stake::total_consensus_stake_key_handle;
+
         use super::*;
 
+        /// Default amount of staked NAM to be used in tests.
+        pub(super) fn default_total_stake() -> token::Amount {
+            // 1000 NAM
+            token::Amount::from(1_000_000_000_u64)
```

Comment:
nit: This doesn't clearly represent the amount of nam. Use the `native_whole` or `from_uint(_, NATIVE_MAX_DECIMAL_PLACES)` instead.